src/main/java/com/cloudant/client/api/ClientBuilder.java

```diff
@@ -0,0 +1,292 @@
+package com.cloudant.client.api;
+
+import static com.cloudant.client.org.lightcouch.internal.CouchDbUtil.assertNotEmpty;
+
+import com.cloudant.client.org.lightcouch.CouchDbException;
+import com.cloudant.client.org.lightcouch.CouchDbProperties;
+import com.cloudant.client.org.lightcouch.internal.URIBuilder;
+import com.cloudant.http.CookieInterceptor;
+import com.cloudant.http.HttpConnectionInterceptor;
+import com.cloudant.http.HttpConnectionRequestInterceptor;
+import com.cloudant.http.HttpConnectionResponseInterceptor;
+import com.cloudant.http.interceptors.ProxyAuthInterceptor;
+import com.cloudant.http.interceptors.SSLCustomizerInterceptor;
+import com.cloudant.http.interceptors.TimeoutCustomizationInterceptor;
+import com.google.gson.GsonBuilder;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import javax.net.ssl.SSLSocketFactory;
+
+public class ClientBuilder {
+
+     /** Default of 5 minutes for connection timeout **/
+    public static final long CONNECTION_TIMEOUT = 5l;
```

Comment:
I don't think these two extra fields are providing much value, can we not just have:
```java
/**Connection timeout defaults to 5 minutes**/
public static final TimeoutOption DEFAULT_CONNECTION_TIMEOUT = TimeoutOption.minutes(5l);
```
and similarly for the read timeout?